js/lesson3/tutorial.md

```diff
@@ -46,7 +50,7 @@ Using jQuery and JavaScript functions, we are going to build a small
 todo list.
 
 Download the files that you will need to work through the example
-[here](download).
+[here](#download-example1). and [example2](#download-example2)
```

Comment:
This is only the test illustration, if you notice the file list is the exercise 1 zip split in 2